scrapers/JavLibrary_python.py

```diff
@@ -174,35 +175,43 @@ def debug(q):
     print(q, file=sys.stderr)
 
 
-def sendRequest(url, head):
-    global maintenance_javlibrary
-    global flag_cloudflare
+def checking_protection(url):
+    url_domain = re.sub(r"www\.|\.com", "", urlparse(url).netloc)
+    debug("[DEBUG] === Checking Status of Javlib site ===")
+    for site in SITE_JAVLIB:
+        url_n = url.replace(url_domain, site)
+        response = requests.get(url_n, headers=JAV_HEADERS, timeout=10)
+        if response.url == "https://www.javlib.com/maintenance.html":
+            debug("[{}] Maintenance".format(site))
+        if "Why do I have to complete a CAPTCHA?" in response.text or "Checking your browser before accessing" in response.text:
+            debug("[{}] Protected by Cloudflare".format(site))
+        elif response.status_code != 200:
+            debug("[{}] Other issue ({})".format(site, response.status_code))
+        else:
+            debug("[{}] Using this site for scraping ({})".format(site, response.status_code))
+            debug("[DEBUG] ======================================")
+            return site, response
```

Comment:
I would move the `response.status_code != 200` check to be the first one, as i am not sure if there will be a `response.text` or a `response.url` if the `status_code` is that of an error.

Reply:
I don't think they can crash anyways:
`response.url` is None by default, since it's a compare it won't crash (`if None == "test"` work)
`response.text` should be same thing.

If i remember correctly the cloudflare page or maintenance is not a status_code 200 so if i put the `response.status_code != 200` first, this condition will always get trigger before other.

Reply:
yeh makes sense now